### PR TITLE
fix(workflows): auto-release

### PR DESCRIPTION
**Pull request by SunsetTechuila**
i'm really sorry

### CODE REVIEW

File: .github/workflows/auto_release.yml [hunk 1]
```diff
@@ -54,6 +54,6 @@ jobs:
       - name: Publish the release
         uses: softprops/action-gh-release@v1
         with:
-          tag_name: "v${{ needs.check_version.outputs.new_version }}"
+          tag_name: ${{ needs.check_version.outputs.new_version }}
```
Comment (theRealPadster) [inline .github/workflows/auto_release.yml:57]: No worries my dude. Don't these need to match though? Looks like it's checking current_version with the v tag and setting the new one to have no v?

Comment (theRealPadster) [reply .github/workflows/auto_release.yml:57]: Or do you mean just do this once and then we change it to just not have the v at all?

Comment (SunsetTechuila) [reply .github/workflows/auto_release.yml:57]: No, right now both variables have "v" before the version number

Comment (theRealPadster) [reply .github/workflows/auto_release.yml:57]: Ohh yes I see, my bad.

**Review by theRealPadster — APPROVED** [review]

**Pull request merged**